tests/genomics_tests/test_genomic_pipeline.py

```diff
@@ -881,6 +883,36 @@ def test_ingest_aw5_file(self, metric_cloud_task):
         self.assertEqual(GenomicSubProcessResult.SUCCESS, self.job_run_dao.get(3).runResult)
         self.assertEqual(GenomicSubProcessResult.SUCCESS, self.job_run_dao.get(4).runResult)
 
+        # Test AW5 Raw table
+        genomic_dispatch.load_manifest_into_raw_table(f"{bucket_name}/{aw5_subfolder_array}/{test_file_name_aw5_array}", "aw5")
+
+        aw5_dao = GenomicDefaultBaseDao(
+            model_type=GenomicAW5Raw
+        )
+        raw_records = aw5_dao.get_all()
+        raw_records.sort(key=lambda x: x.biobank_id)
+
+        with open_cloud_file(os.path.normpath(f"{bucket_name}/{aw5_subfolder_array}/{test_file_name_aw5_array}")) as csv_file:
+            csv_reader = csv.DictReader(csv_file)
+            file_rows = list(csv_reader)
+
+            # Check rows in file against records in raw table
+            i = 0
+            for file_row in file_rows:
```

Comment:
what about `for i, file_row in enumerate(file_rows):`

Reply:
Updated this!